static/des_components/des-y3a2/des-y3a2-gold.html

```diff
@@ -62,9 +100,9 @@ <h3 class="fixdown">Y3_GOLD  &nbsp; &nbsp; <a style="font-size:0.72rem" href="ja
             </tr>
             <tr>
               <td>
-                <a>A_FIDUCIAL_{GRIZY}</a>
+                A_FIDUCIAL_{GRIZY}
               </td>
-              <td>Fiducial reddening correction (subtract off) in band GRIZY [magnitudes]</td>
+              <td>SED-independent interstellar extinction based on the E(B − V) reddening maps from <a href="https://"> Schlegel, Finkbeiner & Davis (1998) </a>  in band GRIZY [magnitudes]</td>
```

Comment:
Is this missing href intentional?

Reply:
That was unintentional. Well spotted! It should be 
https://ui.adsabs.harvard.edu/abs/1998ApJ...500..525S/abstract